news/feat-mamba-dev-testing

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Adding libmamba/libmambapy as development dependencies within docker images (#191)
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* <news item>
+
+### Docs
+
+* <news item>
```

Comment:
We can add here "Added notes about development workflows with libmamba from source" or some other wording.

Reply:
Done.